graylog2-web-interface/src/views/logic/views/UseCreateViewForEvent.ts

```diff
@@ -94,10 +94,9 @@ const createViewWidget = ({ field, groupBy, fnSeries, expr }) => {
 
 const getSummaryAggregation = ({ aggregations, groupBy }) => {
   const { summaryFnSeries, summaryRowPivots, summaryTitle } = aggregations.reduce(
-    (res, { field, value, expr, fnSeries }) => {
+    (res, { value, expr, fnSeries }) => {
       const concatTitle = `${fnSeries} ${expr} ${value}`;
       res.summaryFnSeries.push(fnSeries);
-      if (field) res.summaryRowPivots.push(field);
```

Comment:
we can remove summaryRowPivots from reduce. 
then we can also remove it from `rowPivots: [pivotForField(uniq([...summaryRowPivots, ...groupBy]), new FieldType('value', [], []))],`